playwright/_impl/_playwright.py

```diff
@@ -40,6 +41,15 @@ def __init__(
             for device in initializer["deviceDescriptors"]
         }
 
+    def __getitem__(self, value: str) -> BrowserType:
+        if value == "chromium":
+            return self.chromium
+        elif value == "firefox":
+            return self.firefox
+        elif value == "webkit":
+            return self.webkit
+        raise ValueError(f"Invalid browser {value!r}")
```

Comment:
out of curiosity, what does the !r do again?

Reply:
Prints the browser name in quotes which looks nice in traceback